code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -697,6 +697,7 @@
 
 /datum/outfit/syndicatespace/syndicaptain
```

Comment:
```suggestion
/datum/outfit/syndicatespace/syndicrew/syndicaptain
```

Reply:
And replace outfit var with it in "/obj/effect/mob_spawn/human/syndicatespace/captain" part, at line 673.